packages/dcos-integration-test/extra/test_helpers.py

```diff
@@ -78,7 +78,9 @@ def marathon_test_app(
                 'gracePeriodSeconds': 5,
                 'intervalSeconds': 10,
                 'timeoutSeconds': 10,
-                'maxConsecutiveFailures': 3
+                'maxConsecutiveFailures': 120  # ~20 minutes until restarting
+                # killing the container will rarely, if ever, help this application
+                # reach a healthy state, so do not trigger a restart if unhealthy
```

Comment:
This `maxConsecutiveFailures` bump shouldn't really be needed I think. Because IIRC the health checks don't kick in until `TAST_RUNNING` is actually the state of the task. Which happens only when the executor is able to register with the agent in the specified timeout.